### DISCUSSION TITLE
Student receiving payment notice about copilot

### DISCUSSION THREAD

**Discussion by ttelmo1** [community/community#44511] [GitHub Education]
Before registering as a student on github, I had registered for the 60-day trial of copilot, now as a student this benefit should be free but I have a notice that I will be charged at the end of my trial
Has anyone experienced this?

![problema copilot](https://user-images.githubusercontent.com/108095100/212751347-be91f7bf-98e6-4fc2-a0bc-767567244799.png)

**Comment by NotchL1118** [comment]
the same to me,what should we do?

**Reply by ttelmo1** [reply]
I don`t know, i was expecting someone to reply this

**Comment by schwenkd** [comment]
I had this issue as well. You need to resubscribe to Copilot now that you have student access. Just go to settings -> Copilot in your GitHub account.

I should add that my free trial had fully expired at this point, but I had not opted to pay for continued access. I was getting the message "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended." in VSCode when the trial expired.
